azavea/temperate
# Switch Angular CLI to use sass (.scss)

## Overview

Switches project to compile sass rather than css, since that's what our designers prefer to use.

### Demo

Adding:
```scss
$heading-color: red;

h1, h2 {
    color: $heading-color;
}
```
to `app.component.scss` compiles and displays as expected:

![screen shot 2017-09-25 at 13 51 09](https://user-images.githubusercontent.com/1818302/30823090-1c586614-a1f9-11e7-892f-51b1c4224af7.png)


## Testing Instructions

 * Add scss above to `app.component.scss`
 * Restart dev server, ensure it compiles, and shows demo app page headings in red

Closes #64
